src/SeleniumLibrary/__init__.py

```diff
@@ -223,6 +224,89 @@ class SeleniumLibrary(DynamicCore):
 
     See the `Add Location Strategy` keyword for more details.
 
+    = Browser and Window =
+
+    In this section the concept of Browsers or Webdrivers and its Windows
+    will be explained.
+
+    == Browser ==
+
+    In Selenium Library there are two ways to start a Browser and control it.
+    The first one is the `Open Browser` keyword. This one should be used as
+    default keyword.
+    The second one is the `Create Webdriver` keyword which was useful, when
+    browser options needed to be set. Since Selenium Library 4.0.x `Open Browser`
+    also supports option why `Create Webdriver` will be deprecated.
+    When opening a browser a new Webdriver process is startet and connected to
+    Selenium Library. When using Selenium Grid this browser may be started on any
+    Grid node that has matching capabilities.
+    It is possible to start multiple independent browsers at the same test case.
```

Comment:
I like this chapter, let's keep this in.

Reply:
Did refactor this chapter.
More details on Webdriver and Selenium Stuff.